### DISCUSSION TITLE
请问这里是该点击哪里手动验证的？

### DISCUSSION THREAD

**Discussion by thegame-hhh** [mikumifa/biliTickerBuy#298] [提问专区]
![捕获](https://github.com/mikumifa/biliTickerBuy/assets/174237832/21869da3-1434-4fd6-9bfd-aeb85a20141e)

**Comment by 2536867317** [comment]
有验证码的话会跳出来的，过程项目的GIF好像有

**Reply by thegame-hhh** [reply]
我看了那个gif跟我的好像不太一样。。。

**Reply by 2536867317** [reply]
你这里要是有验证码应该会显示的，但是获取验证码失败，我也不知道

**Comment by mikumifa** [comment]
可能是梯子

**Reply by thegame-hhh** [reply]
可是我开了，关了都这样。。。有时没弹出验证码倒是正常抢

**Reply by thegame-hhh** [reply]
已解决了，是浏览器的问题，我用qq浏览器正常显示，谷歌浏览器不行
